### DISCUSSION TITLE
Actions not working

### DISCUSSION THREAD

**Discussion by xaoyaoo** [community/community#72847] [New to GitHub]
https://github.com/xaoyaoo/PyWxDump/actions/

1. [publishNew.yml](https://github.com/xaoyaoo/PyWxDump/blob/master/.github/workflows/publishNew.yml)

2. https://github.com/xaoyaoo/PyWxDump/actions/workflows/publish.yml

I have deleted the No.2 file, but why are actions still using this file?
I recreated the No.1 file, why doesn't it work?

**Comment by ktauchathuranga** [comment]
workflow have some errors,

![image](https://github.com/community/community/assets/79928743/9c74bf16-39a3-4714-ace1-2614c3f237bf)


i think the problem is in this line, (um not sure)
![image](https://github.com/community/community/assets/79928743/c3bada03-9e39-4851-b4b0-4b9ef66e6d06)

**Reply by xaoyaoo** [reply]
I have modified the yml file, but it is not synchronized. You can check the latest file.
